.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,4 +15,10 @@ greatly reduce the amount of work needed to review your work. -->
 1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
 1. [ ] The changes are tested.
 1. [ ] The code is documented (docstrings, project documentation).
-1. [ ] Changes are mentioned in the changelog.
\ No newline at end of file
+1. [ ] Changes are mentioned in the changelog.
+
+### Test environment config
```

Comment:
Please add a description on what it does, and put a notice that it only works for members